### PR TITLE
:bookmark: version 0.9.0

### PR DESCRIPTION
**Pull request by Mortaro**

### CODE REVIEW

File: src/template/src/Home.jsx [hunk 1]
```diff
@@ -27,41 +27,41 @@ class Home extends Nullstack {
           </Link>
           <h1> {project.name} </h1>
           <p>
-            {{i18n_gettingStarted}}
+            {{ i18n_gettingStarted }}
```
Comment (GuiDevloper) [inline src/template/src/Home.jsx]: This space change is something done by your linter, @Mortaro?
Last time it simply broke the create, pay attention or it going to do it again

Comment (brunolm) [inline src/template/src/Home.jsx:30]:
```suggestion
            {{i18n_gettingStarted}}
```

File: src/template/src/Home.jsx [hunk 2]
```diff
@@ -27,41 +27,41 @@ class Home extends Nullstack {
...
             <Link href="{{PROJECT_SRC}}">
-              {{i18n_srcFolder}}
+              {{ i18n_srcFolder }}
```
Comment (brunolm) [inline src/template/src/Home.jsx:32]:
```suggestion
              {{i18n_srcFolder}}
```

File: src/template/src/Home.jsx [hunk 3]
```diff
@@ -27,41 +27,41 @@ class Home extends Nullstack {
...
             </Link>.
           </p>
           <span>
-            {{i18n_hintExtension}}
+            {{ i18n_hintExtension }}
```
Comment (brunolm) [inline src/template/src/Home.jsx:36]:
```suggestion
            {{i18n_hintExtension}}
```

File: src/template/src/Home.jsx [hunk 4]
```diff
@@ -27,41 +27,41 @@ class Home extends Nullstack {
...
             <Link href="vscode:extension/ChristianMortaro.vscode-nullstack">
-              {{i18n_vsExtension}}
+              {{ i18n_vsExtension }}
```
Comment (brunolm) [inline src/template/src/Home.jsx:38]:
```suggestion
              {{i18n_vsExtension}}
```

File: src/template/src/Home.jsx [hunk 5]
```diff
@@ -27,41 +27,41 @@ class Home extends Nullstack {
...
             </Link>
           </span>
           <ul>
             <li>
               <Link href="{{i18n_link0:0}}">
-                {{i18n_link0:1}}
+                {{ i18n_link0: 1 }}
```
Comment (brunolm) [inline src/template/src/Home.jsx:44]:
```suggestion
                {{i18n_link0: 1}}
```

File: src/template/src/Home.jsx [hunk 6]
```diff
@@ -27,41 +27,41 @@ class Home extends Nullstack {
...
               </Link>
             </li>
             <li>
               <Link href="{{i18n_link1:0}}">
-                {{i18n_link1:1}}
+                {{ i18n_link1: 1 }}
```
Comment (brunolm) [inline src/template/src/Home.jsx:49]:
```suggestion
                {{i18n_link1: 1}}
```

File: src/template/src/Home.jsx [hunk 7]
```diff
@@ -27,41 +27,41 @@ class Home extends Nullstack {
...
               </Link>
             </li>
             <li>
               <Link href="{{i18n_link2:0}}">
-                {{i18n_link2:1}}
+                {{ i18n_link2: 1 }}
```
Comment (brunolm) [inline src/template/src/Home.jsx:54]:
```suggestion
                {{i18n_link2: 1}}
```

File: src/template/src/Home.jsx [hunk 8]
```diff
@@ -27,41 +27,41 @@ class Home extends Nullstack {
...
               </Link>
             </li>
             <li>
               <Link href="{{i18n_link3:0}}">
-                {{i18n_link3:1}}
+                {{ i18n_link3: 1 }}
```
Comment (brunolm) [inline src/template/src/Home.jsx:59]:
```suggestion
                {{i18n_link3: 1}}
```

File: src/template/src/Home.jsx [hunk 9]
```diff
@@ -27,41 +27,41 @@ class Home extends Nullstack {
...
               </Link>
             </li>
             <li>
               <Link href="{{i18n_link4:0}}">
-                {{i18n_link4:1}}
+                {{ i18n_link4: 1 }}
```
Comment (brunolm) [inline src/template/src/Home.jsx:64]:
```suggestion
                {{i18n_link4: 1}}
```

File: src/template/src/Home.tsx [hunk 10]
```diff
@@ -34,41 +33,41 @@ class Home extends Nullstack<HomeProps> {
           </Link>
           <h1> {project.name} </h1>
           <p>
-            {{i18n_gettingStarted}}
+            {{ i18n_gettingStarted }}
```
Comment (brunolm) [inline src/template/src/Home.tsx:36]:
```suggestion
            {{i18n_gettingStarted}}
```

File: src/template/src/Home.tsx [hunk 11]
```diff
@@ -34,41 +33,41 @@ class Home extends Nullstack<HomeProps> {
...
             <Link href="{{PROJECT_SRC}}">
-              {{i18n_srcFolder}}
+              {{ i18n_srcFolder }}
```
Comment (brunolm) [inline src/template/src/Home.tsx:38]:
```suggestion
              {{i18n_srcFolder}}
```

File: src/template/src/Home.tsx [hunk 12]
```diff
@@ -34,41 +33,41 @@ class Home extends Nullstack<HomeProps> {
...
             </Link>.
           </p>
           <span>
-            {{i18n_hintExtension}}
+            {{ i18n_hintExtension }}
```
Comment (brunolm) [inline src/template/src/Home.tsx:42]:
```suggestion
            {{i18n_hintExtension}}
```

File: src/template/src/Home.tsx [hunk 13]
```diff
@@ -34,41 +33,41 @@ class Home extends Nullstack<HomeProps> {
...
             <Link href="vscode:extension/ChristianMortaro.vscode-nullstack">
-              {{i18n_vsExtension}}
+              {{ i18n_vsExtension }}
```
Comment (brunolm) [inline src/template/src/Home.tsx:44]:
```suggestion
              {{i18n_vsExtension}}
```

File: src/template/src/Home.tsx [hunk 14]
```diff
@@ -34,41 +33,41 @@ class Home extends Nullstack<HomeProps> {
...
             </Link>
           </span>
           <ul>
             <li>
               <Link href="{{i18n_link0:0}}">
-                {{i18n_link0:1}}
+                {{ i18n_link0: 1 }}
```
Comment (brunolm) [inline src/template/src/Home.tsx:50]:
```suggestion
                {{i18n_link0: 1}}
```

File: src/template/src/Home.tsx [hunk 15]
```diff
@@ -34,41 +33,41 @@ class Home extends Nullstack<HomeProps> {
...
               </Link>
             </li>
             <li>
               <Link href="{{i18n_link1:0}}">
-                {{i18n_link1:1}}
+                {{ i18n_link1: 1 }}
```
Comment (brunolm) [inline src/template/src/Home.tsx:55]:
```suggestion
                {{i18n_link1: 1}}
```

File: src/template/src/Home.tsx [hunk 16]
```diff
@@ -34,41 +33,41 @@ class Home extends Nullstack<HomeProps> {
...
               </Link>
             </li>
             <li>
               <Link href="{{i18n_link2:0}}">
-                {{i18n_link2:1}}
+                {{ i18n_link2: 1 }}
```
Comment (brunolm) [inline src/template/src/Home.tsx:60]:
```suggestion
                {{i18n_link2: 1}}
```

File: src/template/src/Home.tsx [hunk 17]
```diff
@@ -34,41 +33,41 @@ class Home extends Nullstack<HomeProps> {
...
               </Link>
             </li>
             <li>
               <Link href="{{i18n_link3:0}}">
-                {{i18n_link3:1}}
+                {{ i18n_link3: 1 }}
```
Comment (brunolm) [inline src/template/src/Home.tsx:65]:
```suggestion
                {{i18n_link3: 1}}
```

File: src/template/src/Home.tsx [hunk 18]
```diff
@@ -34,41 +33,41 @@ class Home extends Nullstack<HomeProps> {
...
               </Link>
             </li>
             <li>
               <Link href="{{i18n_link4:0}}">
-                {{i18n_link4:1}}
+                {{ i18n_link4: 1 }}
```
Comment (brunolm) [inline src/template/src/Home.tsx:70]:
```suggestion
                {{i18n_link4: 1}}
```

**Review by brunolm — CHANGES_REQUESTED** [review]

**Review by brunolm — CHANGES_REQUESTED** [review]
https://github.com/nullstack/create-nullstack-app/pull/36 Remove spaces in i18n tags

**Pull request merged**